src/member/javascripts/services/purchase.js.coffee

```
@@ -14,7 +14,7 @@ angular.module('BBMember.Services').factory "MemberPurchaseService", ($q, $rootS
             new BBModel.PurchaseTotal(purchase)
           deferred.resolve(purchases)
         , (err) ->
-          deferred.reject(err)
+          deferred.resolve([])
```

Does the errr object come back with the 404 status? We only want to resolve the promise in this scenario as we might hide genuine errors with this approach.